src/ProvidersCache.js

```diff
@@ -58,7 +58,7 @@ class ProvidersCache {
    * @returns {Promise}
    */
   discover (issuer) {
-    return fetch(`${issuer}/.well-known/openid-configuration`)
+    return fetch(`${issuer}${issuer.endsWith('/') ? '' : '/'}.well-known/openid-configuration`)
```

Comment:
Let's just catch all cases with:
```suggestion
    return fetch(issuer.replace(/\/*$/, '.well-known/openid-configuration')
```